fabfile/run.py

```diff
@@ -203,3 +203,57 @@ def tpch_queries(query_info, connectionURI, pg_version, citus_version, config_fi
         out_val = run(run_string)
         results_file.write(out_val)
         results_file.write('\n')
+
+# Filter (possibly) citus-related valgrind test logs, put under results directory
+# If no error logs exist, then simply put valgrind_success file 
+def filter_put_citus_valgrind_outputs(repo_path):
+    # cd to directory that we performed valgrind(regression) tests
+    os.chdir(os.path.join(repo_path, config.RELATIVE_REGRESS_PATH))
```

Comment:
dont we already use a `cd` wrapper while calling this method, in that case this line can be removed?

Reply:
the `cd` wrapper does not apply to calls made from `os`